cpp11/Ice/interceptor/AuthenticatorI.h

```diff
@@ -12,13 +12,14 @@
 
 class AuthenticatorI : public Demo::Authenticator
 {
+    AuthenticatorI();
+    void validateToken(const std::string&);
+
 public:
 
-    AuthenticatorI();
     virtual Demo::Token getToken(std::string, std::string, const Ice::Current&) override;
-    void validateToken(const std::string&);
 
-    static const long long int TOKEN_LIFETIME = 1000 * 60 * 60;
+    static const long long TOKEN_LIFETIME = 1000 * 60 * 60;
```

Comment:
can we just use a duration `std::chrono::hours(1)` in the implementation and get rid of `TOKEN_LIFETIME`